docs/src/configuration/services/memcached.md

```diff
@@ -18,6 +18,15 @@ Both Memcached and Redis can be used for application caching. As a general rule,
 |----------------------------------|---------------|---------------|
 |  {{< image-versions image="memcached" status="supported" environment="grid" >}} | {{< image-versions image="redis" status="memcached" environment="dedicated" >}} | {{< image-versions image="redis" status="memcached" environment="dedicated-gen-3" >}} |
 
+## Use a framework
+
+If you use one of the frameworks, follow the guide there:
+
+- [Drupal 7](../../frameworks/drupal7/memcached.md)
+- [Drupal 9](../../guides/drupal9/memcached.md)
```

Comment:
```suggestion
- [Drupal](../../guides/drupal9/memcached.md)
- [Drupal 7](../../frameworks/drupal7/memcached.md)
```
Just a possibility :shrug:

Reply:
Drupal 7 was supposed to be EOL last year, so people should not be starting new projects in it. Could conceivably remove its line altogether.